schema.rst

```diff
@@ -429,12 +427,12 @@ use of this attribute is discouraged.
 Specifies the ABI language or languages of a static library
 (`Type`_ :string:`"archive"`).
 Officially supported (case-insensitive) values are
-:string:`"C"` (no special handling required) and
-:string:`"C++"` (consuming the static library
+:string:`"c"` (no special handling required) and
+:string:`"cpp"` (consuming the static library
```

Comment:
This still says case-insensitive

Reply:
Yes, I am still torn on that. For *key* names, there's a technical reason to enforce case-sensitivity. For *values*, doing likewise seems like gratuitous violation of the [Robustness Principle](https://en.wikipedia.org/wiki/Robustness_principle)... which is why I originally specified case-insensitivity in the first place and, if not for said technical issues, would prefer to have retained it.

I'm not saying I'm dead-set on this point, just... reluctant.

Reply:
> Also, I think you could add the Closes: #2 or link the issue.

It was already linked, but good point on adding an auto-close tag; done!